docs/UserGuide.md

```diff
@@ -416,22 +432,23 @@ Example: <br>
 
 * Name is a compulsory field that is case-insensitive but space-sensitive.<br>
 
-* Name must be present in Pooch Planner.<br>
+* Name must be present in PoochPlanner.<br>
 
-* Using pin command on a contact that has been pinned do not make any changes to Pooch Planner.<br>
+* Using pin command on a contact that has been pinned do not make any changes to PoochPlanner.<br>
 
 * A contact will still remain pinned if you use the undo command once on a contact that has been pinned twice or more.<br>
 </div>
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tips:**<br>
 
 * Use the pin command for your frequently contacted addresses! <br>
+* For your convenience, if there are repeated fields, only the value of the latest repeated field will be processed. So, if you have mistakenly entered the wrong value, simply retype the field and value in the same line.<br>
 
 </div>
 
 #### Unpinning a contact : `unpin`
 
-Unpins the specified contact on Pooch Planner so that your less important contacts can be removed from the top on the contact list.
+Unpins the specified contact on PoochPlanner so that your less important contacts can be removed from the top on the contact list.
```

Comment:
Same here, change to "...removed from the top ~~on the~~ _of your_ contact list."

Reply:
changed